File: customers-staff-master/src/Front/src/api/new/dto/ServiceCalendar/ServiceCalendarRecordUpdateDto.ts
import { DateTime } from "../DateTime";
import { Guid } from "../Guid";
import { ServiceCalendarRecordInfoDto } from "./ServiceCalendarRecordInfoDto";

export interface ServiceCalendarRecordUpdateDto extends ServiceCalendarRecordInfoDto {
  updatedDate?: DateTime;
  updatedWorkerId?: Guid;
}

File: customers-staff-master/src/CustomersAndStaff/FrontApi.Dto/Calendars/ServiceCalendarRecordUpdateDto.cs
using System;

namespace Market.CustomersAndStaff.FrontApi.Dto.Calendars
{
    public class ServiceCalendarRecordUpdateDto : ServiceCalendarRecordInfoDto
    {
        public Guid? UpdatedWorkerId { get; set; }
        public DateTime? UpdatedDate { get; set; }
    }
}